src/components/Lesson/lesson.helpers.ts

```diff
@@ -327,3 +331,35 @@ export function groupLessonPathways(pathways: LessonPathway[]) {
     subjects,
   };
 }
+
+type Attribution = {
+  questionNumber: string;
+  attribution: string;
+};
+
+export const createAttributionObject = (
+  questions: LessonOverviewQuizData,
+): Attribution[] | [] => {
+  const attributions: Attribution[] = [];
+  if (questions) {
```

Comment:
can questions be undefined?

Reply:
yeah undefined or null depending on the lesson